src/app/containers/Figure/fixtureData.jsx

```diff
@@ -38,6 +38,7 @@ const captionBlockWithLink = {
                     model: {
                       text: 'containing an inline link',
                       locator: 'https://www.bbc.com',
+                      isExternal: false,
```

Comment:
Any chance we can get a 'true' case in fixture?

Reply:
Added relevant attribute into http://localhost:7080/news/articles/c0000000006o, which had an external link.